website/templates/edit_person.html

```diff
@@ -92,7 +117,33 @@
         </div>
     </div>
     <div class="col ps-2">
-        <input type="text" class="form-control" name="website" value="{{ person.facebook|default('empty') }}">
+        <input type="text" class="form-control" name="facebook" placeholder="Facebook URL"
+            value="{{ socials['facebook']|default('') }}">
     </div>
 </div>
+{% endblock %}
+
+{% block edit_person_password %}
+{# TODO add change emails #}
+<div class="p-3 rounded pt-1">
+</div>
+<div class="p-3 bg-light rounded pt-1">
+    <p class="h4">Edit Account Info</p>
+    <p class="h5">Password</p>
+    <input type="password" class="form-control mb-2" name="current_password" placeholder="Current password">
+    <input type="password" class="form-control my-2" name="new_password" placeholder="Enter new password">
+    <input type="password" class="form-control bt-2" name="confirm_new_password" placeholder="Re-enter new password">
+    <p class="h5 pt-3">Account Email</p>
+    <input type="text" class="form-control mb-2" name="current_login_email" placeholder="Current login email">
+    <input type="text" class="form-control mb-2" name="new_login_email" placeholder="New login email">
+    <input type="text" class="form-control my-2" name="confirm_new_login_email" placeholder="Re-type new login email">
+</div>
+{% endblock %}
+
+{% block edit_person_tags %}
+<div class="form-floating">
+    <textarea class="form-control" placeholder="Enter tags here" name="tags" id="tags"
+        style="height: 35px">{{ tag_name_list|join(',')|default('') }}</textarea>
```

Comment:
If you make this `join(', ')` it will put the spaces between them, instead of adding them in website/routes/ui/people.py:178. Then the first one won't have a space in front of it.